src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -30,7 +30,19 @@ void draw_settings_dialog(HostState &host) {
     ImGuiTabBarFlags settings_tab_flags = ImGuiTabBarFlags_None;
     ImGui::BeginTabBar("SettingsTabBar", settings_tab_flags);
 
+    // Core
+    if (ImGui::BeginTabItem("Core")) {
+        ImGui::PopStyleColor();
+        ImGui::Text("CPU Backend \nSelect your Backend.\n(Reboot after change for apply)");
+        ImGui::RadioButton("Unicorn", &host.cfg.cpu_backend, 0);
+        ImGui::RadioButton("Dynarmic", &host.cfg.cpu_backend, 1);
+        ImGui::EndTabItem();
+    } else {
```

Comment:
Remove this else, popping makes no sense (also if you're gonna do something both in `if` & `else` branches just put it outside of either).

Reply:
Same on my pr you can't touch this is same for all tab